app/views/admins/survey_responses/index.html.erb

```diff
@@ -34,4 +36,8 @@
       </tr>
     <% end %>
   <% end %>
-<% end %>
\ No newline at end of file
+<% end %>
+
```

Comment:
please delete the blank lines at the end of the file

Reply:
Done